api/src/main/java/marquez/db/FlywayFactory.java

```diff
@@ -102,6 +113,7 @@ public Flyway build(@NonNull DataSource source) {
         .placeholderSuffix(placeholderSuffix)
         .sqlMigrationPrefix(sqlMigrationPrefix)
         .repeatableSqlMigrationPrefix(repeatableSqlMigrationPrefix)
+        .schemas(schema.map(s -> new String[] {s}).orElse(new String[] {}))
```

Comment:
Flyway does expose a `defaultSchema`:

> The default schema managed by Flyway. This schema name is case-sensitive. If not specified, but schemas is, Flyway uses the first schema in that list. If that is also not specified, Flyway uses the default schema for the database connection.
Consequences:
> * This schema will be the one containing the schema history table.
> * This schema will be the default for the database connection (provided the database supports this concept).

Would a better option be to set the `defaultSchema`? Right now, we default to the schema defined in the database connection.

Reply:
I think I avoided that because I don't like passing `null` as a method parameter, but I suppose that's what Flyway does internally anyway...

Reply:
Though, I suppose if we're just going to pass in `null`, there's no real point in using `Optional` at all. I just find it distasteful to pass around `null` when better options exist. But, I suppose no point in making things more complicated than they need to be